.github/workflows/deploy-to-production.yml

```diff
@@ -83,7 +83,7 @@ jobs:
   notifications:
       runs-on: ubuntu-latest
       needs: deploy
-      if: always()
+      if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
```

Comment:
Does this mean deploy failures won't be notified on?

Reply:
No it doesn't. It means that if we've skipped the production deployment due to a failing staging run we'll avoid notifications about a skipped run